Carthage/Checkouts/PINCache/Source/PINDiskCache.m

```diff
@@ -343,7 +339,16 @@ - (PINDiskCacheSerializerBlock)defaultSerializer
 - (PINDiskCacheDeserializerBlock)defaultDeserializer
 {
     return ^id(NSData * data, NSString *key){
-        return [NSKeyedUnarchiver unarchiveObjectWithData:data];
+        if (@available(iOS 11.0, macOS 10.13, tvOS 11.0, watchOS 4.0, *)) {
+            NSError *error = nil;
+            NSKeyedUnarchiver *unarchiver = [[NSKeyedUnarchiver alloc] initForReadingFromData:data error:&error];
+            PINDiskCacheLogError(error);
+            unarchiver.requiresSecureCoding = NO;
+            id obj = [unarchiver decodeObjectForKey:NSKeyedArchiveRootObjectKey];
+            return obj;
```

Comment:
Just `return` without the intermediate `obj` variable?

(other occurrences of this below)